src/aks-preview/HISTORY.rst

```diff
@@ -24,6 +24,7 @@ Pending
 * Add `--ssh-access` to the `az aks nodepool add` command.
 * Add `--ssh-access` to the `az aks nodepool update` command.
 * Implicitly enable istio when ingress or egress gateway is enabled for Azure Service Mesh.
+* Add parameter to set revision `--revision` for the Azure Service Mesh addon while creating AKS cluster.
```

Comment:
1.0.0b6/2.0.0b1 has been released, please rebase from main and choose a new version or put the note to pending